src/pages/Payouts/index.tsx

```diff
@@ -58,7 +58,7 @@ export const Payouts = ({ page }: PageProps) => {
               />
             </h4>
             <h2>
-              {fromEra && toEra ? (
+              {fromEra != null && toEra != null ? (
```

Comment:
Can they even be `null`? Seems in [here](https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/blob/570fe711f493f8ddc38f612171fcbd01960a8410/src/pages/Payouts/index.tsx#L127) like it's either `number`, or `undefined`?

Reply:
I use this as a shortcut (notice no double `=`) to check for nullish value
https://stackoverflow.com/a/74974762
I kinda like it for being concise, but I know people have mixed fillings. Let me know, I can change it

Reply:
Yeah, you never know with that one if one made a mistake or used it on purpose :D

I'm not sure myself if I like it, I think I consider this rather a trick, a js backdoor abuse. But if you like it then it's fine.